empress_gui.py

```diff
@@ -0,0 +1,335 @@
+        and closes the window when the checkbox is unchecked.
+        """
+        if self.recon_space_btn_var.get() == 1:
+            self.recon_space_window = tk.Toplevel(self.master)
+            self.recon_space_window.geometry("400x400")
+            self.recon_space_window.title("View reconciliation space")
+            Recon_space_window(self.recon_space_window)
+        if self.recon_space_btn_var.get() == 0:
+            if self.recon_space_window.winfo_exists() == 1:
+                self.recon_space_window.destroy()
+
+    def open_and_close_window_recons(self):
+        """
+        Opens a new window titled "View reconciliations" when the checkbox is checked,
+        and closes the window when the checkbox is unchecked.
+        """
+        if self.recons_btn_var.get() == 1:
+            self.recons_window = tk.Toplevel(self.master)
+            self.recons_window.geometry("400x400")
+            self.recons_window.title("View reconciliations")
+            Recon_window(self.recons_window)
+        if self.recons_btn_var.get() == 0:
+            if self.recons_window.winfo_exists() == 1:
+                self.recons_window.destroy()
+
+    def open_and_close_window_histogram(self):
+        """
+        Opens a new window titled "View p-value histogram" when the checkbox is checked,
+        and closes the window when the checkbox is unchecked.
+        """
+        if self.histogram_btn_var.get() == 1:
+            self.histogram_window = tk.Toplevel(self.master)
+            self.histogram_window.geometry("400x400")
+            self.histogram_window.title("View p-value histogram")
+            Histogram_window(self.histogram_window)
+        if self.histogram_btn_var.get() == 0:
+            if self.histogram_window.winfo_exists() == 1:
+                self.histogram_window.destroy()
+
+### View reconciliation space ###
+class Recon_space_window:
+    def __init__(self, master):
+        self.master = master        
+        self.frame = tk.Frame(master)
+        self.frame.pack(fill=tk.BOTH, expand=1)
+        self.frame.pack_propagate(False)
+
+### View reconciliations ###
+class Recon_window:
+    def __init__(self, master):
+        self.master = master        
+        self.frame = tk.Frame(master)
+        self.frame.pack(fill=tk.BOTH, expand=1)
+        self.frame.pack_propagate(False)
+
+### View p-value histogram ###
+class Histogram_window:
+    def __init__(self, master):
+        self.master = master        
+        self.frame = tk.Frame(master)
+        self.frame.pack(fill=tk.BOTH, expand=1)
+        self.frame.pack_propagate(False)
+
+def on_closing():
+    """Kills the matplotlib program and all other tkinter programs when the master window is closed."""
+    plt.close("all")
+    root.destroy()
+
+
+root = tk.Tk()
+root.geometry("600x600")
+root.title("eMPRess GUI Version 1")
+App(root)
+root.protocol("WM_DELETE_WINDOW", on_closing)
+root.mainloop()
+root.quit()
```

Comment:
Add one trailing new line. (see https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
Done:)